### PR TITLE
CY-1219-import-export-secrets

### PR DESCRIPTION
**Pull request by YoniItzhak**

### CODE REVIEW

File: cloudify/cryptography_utils.py [hunk 1]
```diff
@@ -53,6 +56,22 @@ def _get_encryption_key():
         return rest_security_conf['encryption_key']
 
 
+def generate_key_using_password(password):
```
Comment (adigrabow) [inline cloudify/cryptography_utils.py]: I think `salt` should be a param here (with None as default)

File: cloudify/cryptography_utils.py [hunk 2]
```diff
@@ -53,6 +56,22 @@ def _get_encryption_key():
...
+    password = password.encode()
+    salt = b'salt_'
+    # should verify it is safe enough, option: generate a random one and
```
Comment (adigrabow) [inline cloudify/cryptography_utils.py]: Cab delete this comment

File: cloudify_rest_client/secrets.py [hunk 3]
```diff
@@ -126,6 +133,17 @@ def get(self, key):
...
 
+    def export(self, **kwargs):
+        """
+        Returns a list of secrets to be exported
+
+        :param kwargs: the parameters given from the user
+        :return: Secrets' list
+        """
+        params = kwargs
+        response = self.api.get('/secrets/command/export', params=params)
```
Comment (adigrabow) [inline cloudify_rest_client/secrets.py]: Just change the endpoint name

File: cloudify_rest_client/secrets.py [hunk 4]
```diff
@@ -126,6 +133,17 @@ def get(self, key):
         response = self.api.get('/secrets/{0}'.format(key))
         return Secret(response)
 
+    def export(self, **kwargs):
+        """
+        Returns a list of secrets to be exported
```
Comment (adigrabow) [inline cloudify_rest_client/secrets.py]: Do you think it's clear enough? Why not `exports secrets to file`?

Comment (YoniItzhak) [reply cloudify_rest_client/secrets.py]: I think it's a good idea, but won't it be a bit confusing when all the other functions are named the same as in the cli file?

**Pull request closed without merging**